### ISSUE TITLE
Keyboard Shortcut when you don't know the answer

### ISSUE THREAD

**Issue by morgenman** [Doublevil/Houhou-SRS#36]
Right now, if you don't know the answer, the method for advancing to the answer is space(or any other character) and 'enter'. 
Ideally, hitting the enter key without an answer should work too.

**Comment by Doublevil** [comment]
I disagree. I believe hitting enter when the field is empty should not do anything because it's most likely to be the user mistakenly pressing Enter too many times.
I think having to type anything in and press Enter is fine. I'm not convinced adding a shortcut for that would be very useful, as it's both fast and intuitive as is.
